libcontainer/nsenter/nsenter.go

```diff
@@ -5,8 +5,8 @@ package nsenter
 /*
 #cgo CFLAGS: -Wall
 extern void nsexec();
-void __attribute__((constructor)) init(void) {
-	nsexec();
+void __attribute__((constructor)) init(int argc, char *argv[]) {
+	nsexec(argv);
```

Comment:
And here.

Reply:
We don't really need `argc` but I can pass it if you want to make use of it in the future.

Reply:
Actually, let's remove `argc` from `nsexec()` for now.